commit 327cdb98fb3eed8373777465759c44f8d123c227
Author: Flavio Suligoi <f.suligoi@asem.it>
Date:   Fri Jun 5 17:41:04 2020 +0200

    doc: networking: wireless: fix wiki website url
    
    In the files:
    
    - regulatory.rst
    - mac80211-injection.rst
    
    the wiki url is still the old "wireless.kernel.org"
    instead of the new "wireless.wiki.kernel.org"
    
    Signed-off-by: Flavio Suligoi <f.suligoi@asem.it>
    Link: https://lore.kernel.org/r/20200605154112.16277-2-f.suligoi@asem.it
    Signed-off-by: Johannes Berg <johannes.berg@intel.com>

diff --git a/Documentation/networking/regulatory.rst b/Documentation/networking/regulatory.rst
index 8701b91e81ee..16782a95b74a 100644
--- a/Documentation/networking/regulatory.rst
+++ b/Documentation/networking/regulatory.rst
@@ -9,7 +9,7 @@ regulatory infrastructure works.
 
 More up to date information can be obtained at the project's web page:
 
-http://wireless.kernel.org/en/developers/Regulatory
+https://wireless.wiki.kernel.org/en/developers/Regulatory
 
 Keeping regulatory domains in userspace
 ---------------------------------------
@@ -37,7 +37,7 @@ expected regulatory domains will be respected by the kernel.
 A currently available userspace agent which can accomplish this
 is CRDA - central regulatory domain agent. Its documented here:
 
-http://wireless.kernel.org/en/developers/Regulatory/CRDA
+https://wireless.wiki.kernel.org/en/developers/Regulatory/CRDA
 
 Essentially the kernel will send a udev event when it knows
 it needs a new regulatory domain. A udev rule can be put in place
@@ -58,7 +58,7 @@ Who asks for regulatory domains?
 
 Users can use iw:
 
-http://wireless.kernel.org/en/users/Documentation/iw
+https://wireless.wiki.kernel.org/en/users/Documentation/iw
 
 An example::